From ee5aeb285bc60d9ecefccb85316d631842ec4bc6 Mon Sep 17 00:00:00 2001
From: Torsten Veller <tove@gentoo.org>
Date: Sat, 14 Apr 2012 18:11:54 +0200
Subject: [PATCH 06/30] List packaged patches in patchlevel.h

The list of packaged patches is in patchlevel-gentoo.h, which is generated
from the patches/ directory when building the package.

Taken from Debian.

Bug-Debian: http://bugs.debian.org/567489

Patch-Name: gentoo/patchlevel.diff
---
 patchlevel.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/patchlevel.h b/patchlevel.h
index 1a48368e7c..a6c9b3171a 100644
--- a/patchlevel.h
+++ b/patchlevel.h
@@ -161,6 +161,7 @@ static const char * const local_patches[] = {
         ,"uncommitted-changes"
 #endif
         PERL_GIT_UNPUSHED_COMMITS    	/* do not remove this line */
+#include "patchlevel-gentoo.h"
         ,NULL
 };
 
-- 
2.26.3

